docs/modules/mail-marketing.md

```
@@ -187,57 +280,105 @@ List/detail route modules:
+| `POSTMARK` | `true` + token set | Sends via Postmark with tenant's server token |
 
-Provider behavior:
+New tenants default to `CONSOLE`. Tenant admin sets `provider=POSTMARK` + token via `PUT /api/mail/provider-config`.
 
-- `app.email.provider=console`: mail is bypassed to console sender (dev/stage workflow).
-- Non-console: tenant campaign sends require active tenant provider config (`mail_provider_config`) and encrypted tenant Postmark token.
-- No provider fallback from tenant to platform sender (fail-closed).
+**Console summary logging** — every mail flow logs relevant data at the service layer regardless of provider:
 
-Language assignment policy:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)